### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by CICS-Oleg**

### CODE REVIEW

**Comment by Necr0x0Der** [comment]
I'm not sure in general that these requirements should be in `python` folder of hyperon in general since they are needed only for das-gate

**Review by vsbogd — CHANGES_REQUESTED** [review]
Current list of requirements is kept in `pyproject.toml` file (see `project.dependencies` setting). If we keep das-gate as a part of the `hyperon` Python package then we should keep requirements there. Otherwise we need to distribute it as a separate Python package.

**Comment by Necr0x0Der** [comment]
DAS is invoked as MeTTa extension/module. While it is placed in hyperon package for convenience, putting its requirements into hyperon requirements is not good I think

**Comment by vsbogd** [comment]
> DAS is invoked as MeTTa extension/module. While it is placed in hyperon package for convenience, putting its requirements into hyperon requirements is not good I think

Let's put requirements into a separate file in `python/hyperon/exts/das_gate` directory as discussed.

File: python/requirements.txt
Comment (vsbogd) [inline python/requirements.txt:1]: Please remove this file it doesn't make sense. All dependencies are kept in `pyproject.toml` file.

**Review by vsbogd — APPROVED** [review]

**Pull request merged**